go-controller/pkg/types/const.go

```diff
@@ -187,7 +187,7 @@ const (
 
 	// HandlerSyncTimeout is used when waiting for initial object handler sync.
 	// (i.e. all the ADD events should be processed for the existing objects by the event handler)
-	HandlerSyncTimeout = 20 * time.Second
+	HandlerSyncTimeout = 180 * time.Second
```

Comment:
@ricky-rav can you PTAL here? I thought you mentioned 20 was an ok enough timer.
@cathy-zhou : how many nodes were there in your cluster when you were testing this?

Reply:
the zone we tried has over 800 nodes, and we do have zones of nodes more than 1000. @ricky-rav are you going to port your fixes here?

Reply:
@ricky-rav with your optimization fix, it still takes 50 seconds for our 800+ nodes zone.

Reply:
ok, sold :) Do you also know how many services you have?

Reply:
not many, 80+